response_test.go

```diff
@@ -22,6 +22,7 @@ func TestResponseWriter(t *testing.T) {
 	r.Write([]byte("boop"))
 	b, _ := r.BodyBytes(true)
 	assert.Equal(t, []byte("boop"), b)
+	assert.Equal(t, r.ContentLength, int64(4))
```

Comment:
nitpick but can we follow the semantics of `expected`, `actual` (flip the two arguments). This also holds for the other test cases added 🙏